pages/blog/escrow.js

```diff
@@ -0,0 +1,198 @@
+            </p>
+          </section>
+          <section className="blog__section">
+            <h3>How escrow financial transaction works</h3>
+            <p>
+              Initially, either the payee or the payer approaches Escrow which
+              acts as an arbitrator, the terms and conditions should be agreed
+              before the transaction is started.
+              <ul className="blog__list">
+                <li>
+                  The buyer puts the payment to escrow to purchase a product or
+                  service.
+                </li>
+                <li>
+                  Then, the seller ships the product or the service to the
+                  buyer.
+                </li>
+                <li>
+                  If everything works well, the buyer releases the funds to the
+                  seller. If not, a dispute can be created and the arbitrator
+                  will solve it.
+                </li>
+              </ul>
+            </p>
+          </section>
+          <section className="blog__section">
+            <h3>What is an Escrow Smart contract?</h3>
+            <p>
+              A smart contract is a program that is smart enough to execute a
+              piece of code instructions when conditions are met. Also, smart
+              contracts do not require any third-party control and run solely on
+              the basis of pre-programmed logic which is overseen by a
+              distributed, decentralized network of computers that runs on
+              <a
+                className="blog_link"
+                href="https://www.investopedia.com/terms/b/blockchain.asp#what-is-blockchain"
+              >
+                blockchain
+              </a>
+              . It is a computer protocol that is used to digitally and
+              automatically facilitate, verify, and enforce the terms of a
+              contract. The main benefit of the Escrow smart contract is to
+              reduce the dependency of intermediates, centralized arbitrators to
+              help reduce fraud transactions. The program automatically
+              validates a condition and determines whether an asset will go to
+              one person or back to the person who sent it. It is interesting to
+              know that during the process, the
+              <a
+                className="blog_link"
+                href="https://searchcio.techtarget.com/definition/distributed-ledger"
+              >
+                distributed ledger
+              </a>{" "}
+              can also store and make a copy of the document which provides
+              added security and immutability.
+            </p>
+          </section>
+          <section className="blog__section">
+            <h3>Main Advantages of using Escrow Smart Contracts</h3>
+            <div>
+              <p>
+                <u>
+                  Main Advantages of using Escrow Smart Contracts Smart Contract
```

Comment:
@bhavesh92pandey we have to precise for "escrow". And it will better to add to convert each points to titles.

Reply:
Okay. We can make the changes in the Blog